astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+  lang="c#" />
+
+Copy the following code into `Index.cshtml` and notice the logic to show a different view to users that have not logged in.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml"}
+  lang="razor" />
+
+#### Styling
+To give the pages the look and feel of an application, apply the following styling and images.
+
+Add the `changebank.css` file in the `wwwroot\css" directory of the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/wwwroot/css/changebank.css"}
+  lang="css" />
+
+Add the Changebank logo to the application.
+
+Copy the money image for the `Index` page.
+
+```shell
+curl --create-dirs --output wwwroot/img/money.jpg  https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-web/main/complete-app/app/assets/images/money.jpg
+```
+
+## Conclusion
+
+Your directory for the `complete-application` tree should look like this:
+
+```text
+└── complete-application
+    ├── appsettings.Development.json
+    ├── appsettings.json
+    ├── bin/
+        ├── ...
+    ├── obj/
+        ├── ...
+    ├── Pages/
+    │   ├── _ViewImports.cshtml
+    │   ├── _ViewStart.cshtml
+    |   |── Account.cshtml
+    |   |── Accuont.cshtml.cs
+    │   ├── Error.cshtml
+    │   ├── Error.cshtml.cs
+    │   ├── Index.cshtml
+    │   ├── Index.cshtml.cs
+    │   ├── Logout.cshtml
+    │   ├── Logout.cshtml.cs
+    |   |── MakeChange.cshtml
+    |   |── MakeChange.cshtml.cs
+    │   ├── Privacy.cshtml
+    │   ├── Privacy.cshtml.cs
+    │   └── Shared/
+    │       ├── _Layout.cshtml
+    │       ├── _Layout.cshtml.css
+    │       └── _ValidateScriptsPartial.cshtml
+    ├── Program.cs
+    ├── Properties/
+        ├── ...
+    ├── Startup.cs
+    └── wwwroot/
+        ├── ...
+```
+
+Once you've created this directory structure, you can start up the {frontmatter.technology} application using this command:
+
+```shell
+cd complete-application
+dotnet publish -r osx-x64
+dotnet run
+```
+
+The full code is available [here](https://github.com/FusionAuth/fusionauth-quickstart-dotnet-web).
+
+## Troubleshooting
+
+If you run into an issue with cookies on Safari or other browsers, you might need to run the ASP.NET application under SSL.
+
+You can follow [this guide](https://learn.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide) to install development SSL certificates for your .NET environment.
+
+Alternatively, you can run the project using [Visual Studio](https://visualstudio.microsoft.com), which will run the project using SSL.
+
+If you do this, make sure to update the `Authorized Redirect URL` to reflect the `https` protocol. Also note that the project will probably run on a different port when using SSL, so you must update that as well. To do so, log into the administrative user interface, navigate to `Applications`, then click the `Edit` button on your application and navigate to the `OAuth` tab. You can have more than one URL.
+
```

Comment:
Got an exception on logout.
<img width="993" alt="Screenshot 2023-08-22 at 2 54 03 PM" src="https://github.com/FusionAuth/fusionauth-site/assets/91825/637e0db6-ee8d-4a29-97fb-aa4a910d670b">

Reply:
That was with firefox on mac and safari on mac.

Reply:
I think this was because the section in the configuration file was named `complete-application` and then since you changed the name of the app it did not read.  When I redid a few things I updated that section to contain a more static name.  This should be good now.

Reply:
ah, great!